packages/fxa-settings/src/components/Settings/DropDownAvatarMenu/index.tsx

```diff
@@ -64,7 +64,7 @@ export const DropDownAvatarMenu = () => {
           aria-label={dropDownMenuTitle}
           aria-expanded={!!isRevealed}
           aria-haspopup="menu"
-          className="rounded-full border border-transparent hover:border-purple-500 focus:border-purple-500 focus:outline-none active:border-purple-700 transition-standard"
+          className="rounded-full border border-transparent hover:border-purple-500 active:border-purple-700 transition-standard focus-visible-neutral focus-visible:border-transparent"
```

Comment:
You're already aware of this, but just noting here that the hover/active states here need a review from UX because they don't look clean when combined with the focus outline. This could be pulled in as a quick follow-up with some of the other interaction state updates you have planned (e.g., for the shadowy outlines on links) instead of holding up this PR though.